examples/sensortag.py

```diff
@@ -98,7 +98,7 @@
 
 
 async def main(address):
-    async with BleakClient(address) as client:
+    async with BleakClient(address, win={"use_cached_services": True}) as client:
```

Comment:
I don't have strong feelings about it but I kind of like `win=dict(use_cached_services=True)` to use the kwargs syntax for the `win` args too.

Reply:
I am not sure I understand. Do you mean only document them in the docstrings and not specify them in the method parameters? Still let them enter methods in the `kwargs`? Or do you mean how the dict is constructed with a `=` instead of a `:`?

Reply:
I just mean how the dict is constructed.

Reply:
Ok. Changing.